DataConnectors/Digital shadows/digitalshadowsARM.json

```diff
@@ -161,10 +164,10 @@
                         "WorkspaceID": "[parameters('WorkspaceID')]",
                         "WorkspaceKey": "[parameters('WorkspaceKey')]",
                         "digitalshadowsAccountID": "[parameters('digitalshadowsAccountID')]",
-                        "digitalshadowsUrl": "[parameters('digitalshadowsUrl')]",
+                        "historicaldays": "[parameters('historicaldays')]",
                         "digitalshadowsKey": "[parameters('digitalshadowsKey')]",
                         "digitalshadowsSecret": "[parameters('digitalshadowsSecret')]",
-                        "WEBSITE_RUN_FROM_PACKAGE": "https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/digital%20shadows/digitalshadowsConnector.zip"
+                        "WEBSITE_RUN_FROM_PACKAGE": "https://github.com/metron-labs/Azure-Sentinel/raw/master/DataConnectors/Digital%20shadows/digitalshadows.zip"
```

Comment:
digitalshadowsConnector is better name, why did u change it?

Reply:
I will rename it, I must have changed it during testing the code.